workspace/__ardulib__/Graphics/Text.h

```diff
@@ -0,0 +1,37 @@
+
+#ifndef TEXT_H
+#define TEXT_H
+
+#include "XGraphics.h"
+#include "Fonts/glcdfont.h"
+
+/*
+ * Represents a text given by a pivot point (x, y) which is the top left point of the text,
+ * maximum text length and the font scale size.
+ * By now this class uses only the standard 5x7 GLCD ASCII font.
+ * The `setStyle(XColor*) method sets up text color otherwise
+ * the default foreground color of the scene is used.
+ * The `setStyle(uint8_t) method sets up the font size (scaling the font).
+ */
+class Text : public XGraphics {
+private:
+    XVector2<int16_t> _pivot;
+    char* _text;
+    size_t _textLen;
+
+    XColor* _textColor = nullptr;
+    uint8_t _textSize = 1;
+
+public:
+    Text(XGraphics* parent, char* text, size_t textLen);
+
+    void setPosition(int16_t x, int16_t y);
+    void setStyle(XColor* textColor);
+    void setStyle(uint8_t textSize);
```

Comment:
And what if we’ll want to add letter-spacing or line-height later? Would you add the following?

```cpp
void setStyle(uint8_t letterSpacing);
void setStyle(uint8_t lineHeight);
```

OOops. Ambiguous overload error: `setStyle(4)`. That’s why I’m sure calling everyting `setStyle` instead of `setColor`, `setWeight`, `setTextSize`, `setLineHeight`, etc, etc is a bad idea. A slightly better decision would be:

```cpp
setStyle(XColor* textColor, uint8_t textSize, uint8_t letterSpacing, uint8_t lineHeight /* , ... */);
```

But it does not scale well.

Suggest clear names that reflect what’s going been set.